elasticjob-error-handler/elasticjob-error-handler-dingtalk/src/main/java/org/apache/shardingsphere/elasticjob/error/handler/env/DingtalkEnvironment.java

```diff
@@ -0,0 +1,117 @@
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Bootstrap env.
+ */
+@Slf4j
+public final class DingtalkEnvironment {
+    
+    @Getter
+    private static final DingtalkEnvironment INSTANCE = new DingtalkEnvironment();
+    
+    private static final String PROPERTIES_PATH = "conf/dingtalk.properties";
+    
+    private final Properties properties;
+    
+    private DingtalkEnvironment() {
+        properties = getProperties();
+    }
+    
+    private Properties getProperties() {
+        Properties result = new Properties();
+        try (InputStream fileInputStream = this.getClass().getClassLoader().getResourceAsStream(PROPERTIES_PATH)) {
+            if (fileInputStream != null) {
+                result.load(fileInputStream);
+            }
+        } catch (final IOException ex) {
+            log.warn("Can not load properties file from path: '{}'.", PROPERTIES_PATH);
+        }
+        setPropertiesByEnv(result);
+        return result;
+    }
+    
+    private void setPropertiesByEnv(final Properties prop) {
+        for (EnvironmentArgument each : EnvironmentArgument.values()) {
+            String key = each.getKey();
+            String value = System.getProperties().getProperty(key);
+            if (!Strings.isNullOrEmpty(value)) {
+                log.info("Load property {} with value {} from ENV.", key, value);
+                prop.setProperty(each.getKey(), value);
+            }
+        }
+    }
+    
+    /**
+     * Get dingtalk configuration.
+     *
+     * @return dingtalk configuration
+     */
+    public DingtalkConfiguration getDingtalkConfiguration() {
+        String webhook = getValue(EnvironmentArgument.WEBHOOK);
+        String keyword = getValue(EnvironmentArgument.KEYWORD);
+        String secret = getValue(EnvironmentArgument.SECRET);
+        String connectTimeout = getValue(EnvironmentArgument.CONNECT_TIMEOUT);
+        String readTimeout = getValue(EnvironmentArgument.READ_TIMEOUT);
+        return new DingtalkConfiguration(webhook, keyword, secret, Integer.parseInt(connectTimeout), Integer.parseInt(readTimeout));
+    }
+    
+    private String getValue(final EnvironmentArgument environmentArgument) {
+        String result = properties.getProperty(environmentArgument.getKey(), environmentArgument.getDefaultValue());
+        if (environmentArgument.isRequired()) {
+            Preconditions.checkState(!Strings.isNullOrEmpty(result), String.format("Property `%s` is required.", environmentArgument.getKey()));
+        }
+        return result;
+    }
+    
+    /**
+     * Env args.
+     */
+    @RequiredArgsConstructor
+    @Getter
+    public enum EnvironmentArgument {
+        
+        WEBHOOK("webhook", "", true),
+        
+        KEYWORD("keyword", "", false),
+        
+        SECRET("secret", "", false),
+        
+        CONNECT_TIMEOUT("connectTimeout", "3000", false),
+        
+        READ_TIMEOUT("readTimeout", "5000", false);
```

Comment:
I think the name of keys are too simple that may increase the probability of conflict. Maybe we can define a prefix to prevent conflict. What do you think?